pkg/identityserver/store/errors.go

```diff
@@ -124,4 +124,8 @@ var (
 		"application_dev_eui_limit_reached",
 		"application issued DevEUI limit ({dev_eui_limit}) reached",
 	)
+
+	ErrContactInfoRestricted = errors.DefinePermissionDenied(
```

Comment:
Maybe the description should be `contact information can only reference to the caller` ?

Reply:
Apparently it should be `contact information can only reference the caller`, apologies.